assets/css/events.css

```diff
@@ -110,3 +111,15 @@
     object-fit: cover;
     position: center;
 }
+
+/* Mobile */
+@media screen and (max-width: 736px) {
```

Comment:
If you make the browser window very thin (phone-like) then make it wide again, these changes remain. Please update the `/* Widescreen */` changes to override the `/* Mobile */` changes.

Reply:
This does not occur to me?